marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx

```diff
@@ -564,12 +564,14 @@ const ResultsTabularView = (props) => {
   return (
     <>
       <div className={styles.icon}>
-        <div className={styles.queryExport} data-cy="query-export">
-          {canExportQuery && searchOptions.entityTypeIds.length > 0 && <QueryExport hasStructured={props.hasStructured} columns={props.columns} selectedPropertyDefinitions={props.selectedPropertyDefinitions} />}
-        </div>
-        {searchOptions.entityTypeIds?.length === 1 ? <div className={styles.columnSelector} data-cy="column-selector">
-          <ColumnSelector popoverVisibility={popoverVisibility} setPopoverVisibility={setPopoverVisibility} entityPropertyDefinitions={props.entityPropertyDefinitions} selectedPropertyDefinitions={props.selectedPropertyDefinitions} setColumnSelectorTouched={props.setColumnSelectorTouched} columns={props.columns} primaryKey={primaryKey} />
-        </div> : ""}
+        {searchOptions.entityTypeIds?.length === 1 ? <>
+          <div className={styles.queryExport} data-cy="query-export">
+            {canExportQuery && searchOptions.entityTypeIds.length > 0 && <QueryExport hasStructured={props.hasStructured} columns={props.columns} selectedPropertyDefinitions={props.selectedPropertyDefinitions} />}
```

Comment:
The condition "searchOptions.entityTypeIds.length > 0" seems unnecessary here since we have already restricted the length of "searchOptions.entityTypeIds" to 1 in line 567.

Reply:
fixed